pkg/storaged/nfs-details.jsx

```diff
@@ -41,10 +44,22 @@ function nfs_busy_dialog(client,
             $('#error-popup-message').text(error.toString());
             $('#error-popup').modal('show');
         } else {
+            let sessions = [ ], services = [ ];
+            users.forEach((u) => {
+                var since = moment.duration(-u.since*1000).humanize(true);
```

Comment:
For the future, `let` :-) (but here `since` is the only place where it's being used, so there's no risk of namespace collision).